### DISCUSSION TITLE
How to know if "Full Node Synced" really means synced

### DISCUSSION THREAD

**Discussion by FSys77** [Chia-Network/chia-blockchain#5622] [General]
I have two Chia installations. One on premise and one in the Google Cloud. Both mostly show the same block height and both **always** show "Full Node Synced". Which is a bit strange, because the installation in the Google Cloud is sometimes 2, 3 or even 10 blocks ahead of my local installation.

So how can both installations show "Full Node Synced"? This can't be correct.

How to figure out if a node is **truly** synced with the network?

**Comment by chiabug** [comment]
They are both synced as far as they know from the height sent via the peers they are connected to.  In their own corner of the virtual world they both think they are fully synced.  Your point stands though because obviously the one that is actually "behind" will not be capable of answering a challenge and creating an acceptable proof.
